src/lem/interpreter.rs

```diff
@@ -134,29 +166,78 @@ impl Block {
                     let a = bindings.get(a)?;
                     let b = bindings.get(b)?;
                     let c = match (a, b) {
-                        (Ptr::Leaf(Tag::Expr(Num), f), Ptr::Leaf(Tag::Expr(Num), g)) => {
+                        (Ptr::Leaf(_, f), Ptr::Leaf(_, g)) => {
                             Ptr::Leaf(Tag::Expr(Num), *f * g.invert().unwrap())
```

Comment:
This should return an explicit Lurk error if `g` is 0.

Reply:
Or, I guess that's true of Lurk evaluation — but not what's happening here. As long as this behavior doesn't interfere with that, fine… I would probably at least add an `expect` with an error message signaling the reason for failure. Probably it shouldn't be possible to induce panics this way even in LEM, but if LEM has no error handling, maybe it's fine. It does mean that LEM programmers will just *always* have to guard their usage of division, though, unless they have static guarantees that `g != 0`.

Reply:
I've added a `bail!` with the appropriate message. The interpreter doesn't need to panic there. It's not like the interpreter itself has a correctness error that justifies a panic. It's just that it's interpreting code that allowed that to happen. Whoever calls the interpreter handles the error.